web/libs/storybook/tests/support/e2e.ts

```diff
@@ -0,0 +1,24 @@
+import { CURRENT_FLAGS } from '../feature-flags';
+import '@humansignal/frontend-test/cypress/support/e2e'
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  //it is to prevent outliner failing tests with uncaught error
+  return false;
+});
+
+beforeEach(() => {
+  cy.on('uncaught:exception', err => {
+    return !err.message.includes('ResizeObserver loop completed with undelivered notifications.');
```

Comment:
Most probably we have some other errors, I remember @Gondragos added similar ignore procedure for them. And potentially we could have such error, so better to use array of errors to ignore.

Reply:
ty, reorganized the setup here. I also removed whole e2e section as it's the same thing in cypress. so we only have components tests there now